src/main/java/org/folio/dew/service/BulkEditItemContentUpdateService.java

```diff
@@ -118,8 +119,16 @@ private ContentUpdateRecords applyContentUpdates(List<ItemFormat> itemFormats, C
         result.addToUpdated(updatedItemFormat);
         result.addToPreview(updatedItemFormat);
       } else {
-        result.addToPreview(itemFormat);
-        errorMessage.setValue(NO_CHANGE_MESSAGE);
+        var statusUpdate = contentUpdates.getContentUpdates().stream()
+            .filter(contentUpdate -> contentUpdate.getOption() == STATUS)
+            .findFirst();
+        if (statusUpdate.isPresent() && nonNull(statusUpdate.get().getValue()) && !extractStatusName(itemFormat.getStatus()).equals(statusUpdate.get().getValue())) {
```

Comment:
I guess NO_CHANGE_MESSAGE should be set up in any case here.

Reply:
No, NO_CHANGE_MESSAGE should be set only in case when statuses are equal.